src/ert/gui/ertwidgets/customdialog.py

```diff
@@ -13,39 +15,47 @@
 class CustomDialog(QDialog):
     INVALID_COLOR = QColor(255, 235, 235)
 
-    def __init__(self, title="Title", description="Description", parent=None):
+    def __init__(
+        self,
+        title: str = "Title",
+        description: str = "Description",
+        parent: Optional[QWidget] = None,
+    ) -> None:
         QDialog.__init__(self, parent)
 
-        self._option_list = []
-        """ :type: list of QWidget """
+        self._option_list: List[QWidget] = []
 
         self.setModal(True)
         self.setWindowTitle(title)
 
-        self.layout = QFormLayout()
-        self.layout.setFieldGrowthPolicy(QFormLayout.ExpandingFieldsGrow)
-        self.layout.setSizeConstraint(QLayout.SetFixedSize)
+        self._layout = QFormLayout()
+        self._layout.setFieldGrowthPolicy(
+            QFormLayout.FieldGrowthPolicy.ExpandingFieldsGrow
+        )
+        self._layout.setSizeConstraint(QLayout.SizeConstraint.SetFixedSize)
 
         label = QLabel(description)
-        label.setAlignment(Qt.AlignHCenter)
+        label.setAlignment(Qt.AlignmentFlag.AlignHCenter)
 
-        self.layout.addRow(self.createSpace(5))
-        self.layout.addRow(label)
-        self.layout.addRow(self.createSpace(10))
+        self._layout.addRow(self.createSpace(5))
```

Comment:
Why are you renaming `layout`?

Reply:
it shadows QDialog.layout so mypy complains.